.github/workflows/generate.yml

```diff
@@ -0,0 +1,64 @@
+name: Generate
+
+on:
+  workflow_dispatch:
+    inputs:
+      kubernetesBranch:
+        type: string
+        required: true
+        description: 'The remote kubernetes release branch to fetch openapi spec. .e.g. "release-1.23"'
+
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Java
+        uses: actions/checkout@v3
+      - name: Checkout Gen
+        uses: actions/checkout@v3
+        with:
+          path: gen
+          repository: kubernetes-client/gen
+          token: ${{ secrets.PAT_TOKEN }}
```

Comment:
I don't think we need a token do we? This should be fully public.

Reply:
the default token doesnt allow any cross-repo read/write even tho the repo is public. so we have to use a PAT token here, i granted the token merely read access